src/ppom-controller.ts

```diff
@@ -769,12 +771,16 @@ export class PPOMController extends BaseControllerV2<
           Number(this.state.chainStatus[c2]?.lastVisited) -
           Number(this.state.chainStatus[c1]?.lastVisited),
       )[NETWORK_CACHE_LIMIT.MAX];
-      oldChaninIds.push(oldestChainId);
+      if (oldestChainId) {
+        oldChaninIds.push(oldestChainId);
+      }
     }
 
     const chainStatus = { ...this.state.chainStatus };
     oldChaninIds.forEach((chainId) => {
-      delete chainStatus[chainId];
+      if (chainId) {
+        delete chainStatus[chainId];
+      }
```

Comment:
would `delete chainStatus[chainId!]` work instead? these should always have a value

Reply:
This became unnecessary with `oldestChainIds` being guaranteed as `string[]`. 
Removed here: https://github.com/MetaMask/ppom-validator/pull/89/commits/fe306604f3be5f18d90b77b5175f451fd7073c97